Broken with latest wlroots

Probably related to this: https://github.com/swaywm/wlroots/pull/1387

```
FAILED: bspwc/bspwc@@bspwc@exe/server.c.o
/nix/store/10yq7kwlvbc6h658izmrlsspry1g9f3c-gcc-wrapper-7.3.0/bin/cc -Ibspwc/bspwc@@bspwc@exe -Ibspwc -I../bspwc -I../include -I/nix/store/w5r71zav1c4zmbi3p902bvl7b1zmyqyc-wayland-1.16.0/include -I/nix/store/yayjdd7fsghs2j1rm9rxsm99v4xd0k8m-wlroots-05bb44078667a11216611a8fbe87c5fe69e117f8/include -I/nix/store/lmv7cpljbqzbn0mxfpnk2knsm4c6bq12-pixman-0.34.0/include/pixman-1 -I/nix/store/7yd4f3pqs30lzixja3gw0bxv2jqdkr2d-libxkbcommon-0.8.0-dev/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -O3 -Wno-unused-parameter -Wextra -DWLR_USE_UNSTABLE -D_FORTIFY_SOURCE=2  -MD -MQ 'bspwc/bspwc@@bspwc@exe/server.c.o' -MF 'bspwc/bspwc@@bspwc@exe/server.c.o.d' -o 'bspwc/bspwc@@bspwc@exe/server.c.o' -c ../bspwc/server.c
In file included from ../include/bspwc/server.h:18:0,
                 from ../bspwc/server.c:1:
../include/bspwc/backend.h:11:10: fatal error: wlr/types/wlr_primary_selection.h: No such file or directory
 #include <wlr/types/wlr_primary_selection.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/bspwc/backend.c
+++ b/bspwc/backend.c
@@ -22,7 +22,7 @@ struct backend* create_backend(struct server* server)
 		);
 	backend->wlr_gamma_control_manager = wlr_gamma_control_manager_create(server->display);
 	backend->wlr_screenshooter = wlr_screenshooter_create(server->display);
-	backend->wlr_primary_selection_device_manager = wlr_primary_selection_device_manager_create(server->display);
+	backend->wlr_gtk_primary_selection_device_manager = wlr_gtk_primary_selection_device_manager_create(server->display);
 	backend->wlr_idle = wlr_idle_create(server->display);
 	backend->wlr_idle_inhibit = wlr_idle_inhibit_v1_create(server->display);
 	backend->wlr_linux_dmabuf = wlr_linux_dmabuf_v1_create(server->display, wlr_backend_get_renderer(backend->wlr_backend));

--- a/include/bspwc/backend.h
+++ b/include/bspwc/backend.h
@@ -8,7 +8,7 @@
 #include <wlr/types/wlr_idle.h>
 #include <wlr/types/wlr_idle_inhibit_v1.h>
 #include <wlr/types/wlr_linux_dmabuf_v1.h>
-#include <wlr/types/wlr_primary_selection.h>
+#include <wlr/types/wlr_gtk_primary_selection.h>
 #include <wlr/types/wlr_screenshooter.h>
 #include <wlr/types/wlr_server_decoration.h>
 #include <wlr/types/wlr_xdg_shell.h>
@@ -32,8 +32,8 @@ struct backend
 	struct wlr_gamma_control_manager *wlr_gamma_control_manager;
 	struct wlr_screenshooter *wlr_screenshooter;
 	struct wlr_server_decoration_manager *wlr_server_decoration_manager;
-	struct wlr_primary_selection_device_manager 
-		*wlr_primary_selection_device_manager;
+	struct wlr_gtk_primary_selection_device_manager 
+		*wlr_gtk_primary_selection_device_manager;
 	struct wlr_idle *wlr_idle;
 	struct wlr_idle_inhibit_manager_v1 *wlr_idle_inhibit;
 	struct wlr_linux_dmabuf_v1 *wlr_linux_dmabuf;

